addons/loyalty/models/loyalty_rule.py

```diff
@@ -60,9 +60,13 @@ def _get_reward_point_mode_selection(self):
 
     minimum_qty = fields.Integer('Minimum Quantity', default=1)
     minimum_amount = fields.Monetary('Minimum Purchase', 'currency_id')
-    minimum_amount_tax_mode = fields.Selection([
-        ('incl', 'Included'),
-        ('excl', 'Excluded')], default='incl', required=True,
+    minimum_amount_tax_mode = fields.Selection(
+        selection=[
+            ('incl', 'tax included'),
+            ('excl', 'tax excluded'),
```

Comment:
```suggestion
            ('incl', "tax included"),
            ('excl', "tax excluded"),
```
team guidelines: double quoted strings for strings shown to users.